suite-native/config/src/build.ts

```diff
@@ -0,0 +1,7 @@
+export const getAppVersion = () => process.env.VERSION || '';
```

Comment:
not sure about name of this file, I would call it something like `debug/devUtils`